### DISCUSSION TITLE
notifications (push/email) for available app updates

### DISCUSSION THREAD

**Discussion by elpeh** [nextcloud/all-in-one#3624] [Questions] [answered]
Hello,

NC setup within AIO (now 7.5.1 with 27.1.3, but behaviour since installation some month ago) shows no updates section in the administration/overview settings. This is probably intended because NC update is done within AIO.
But also missing is the setting regarding groups of users that should get update notifications as I have in a "classic" installation.
As a consequence there are no notifications regarding availible app updates. Only visible in the apps section as admin user itself, but I'd like to have mails... I looked for activating via config.php, but no success.

Regards, Lutz

**Comment by szaimen** [comment, marked as answer]
Hi, we are not going to add the updatenotification app back because it shows wrong update information.

As a workaround you might want to enable daily backups which also allows to update the included Nextcloud apps on a weekly basis which should eliminate the need for getting update notifications about available app updates imho.
